Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -51,5 +62,24 @@ def symlink_ld_so
       FileUtils.ln_sf ld_so, brew_ld_so
     end
     private_class_method :symlink_ld_so
+
+    def symlink_gcc_libs
+      gcc_opt_prefix = HOMEBREW_PREFIX/"opt"/OS::LINUX_PREFERRED_GCC_FORMULA
+
+      GCC_RUNTIME_LIBS.each do |library|
+        gcc_library = gcc_opt_prefix/"lib/gcc"/GCC_VERSION_SUFFIX/library
```

Comment:
```suggestion
        gcc_library = gcc_opt_prefix/"lib/gcc/#{GCC_VERSION_SUFFIX}/#{library}"
```

Reply:
I prefer leaving out the interpolation, as it's much less readable.

Reply:
I disagree. It's also less elegant as it's creating a new `Pathname` for each `/`.